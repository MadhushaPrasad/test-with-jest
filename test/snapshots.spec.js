const user = {
  name: "Madhusha Prasad",
  age: 25,
  job: "Software Engineer",
  address: {
    city: "Colombo",
    country: "Sri Lanka",
  },
};

test("User matches the snapshot", () => {
    const userString = `{
                "name": "Madhusha Prasad",
                "age": 25,
                "job": "Software Engineer",
                "address": {
                        "city": "Colombo",
                        "country": "Sri Lanka"
                }
        }`;

    expect(JSON.stringify(user)).toMatchSnapshot(userString);
});
