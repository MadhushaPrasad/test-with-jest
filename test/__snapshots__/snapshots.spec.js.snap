// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User matches the snapshot: {
                "name": "Madhusha Prasad",
                "age": 25,
                "job": "Software Engineer",
                "address": {
                        "city": "Colombo",
                        "country": "Sri Lanka"
                }
        } 1`] = `"{"name":"Madhusha Prasad","age":25,"job":"Software Engineer","address":{"city":"Colombo","country":"Sri Lanka"}}"`;
